file-mon/src/main/java/com/wipro/ats/bdre/filemon/FileMonRunnableMain.java

```diff
@@ -108,8 +108,6 @@ public static void main(String[] args) {
 
     private void execute(String[] params) {
         try {
-            CommandLine commandLine = getCommandLine(params, PARAMS_STRUCTURE);
-
             GetProcess getProcess = new GetProcess();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "commandLine" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1481)